frontend/src/queries/Query/Query.tsx

```diff
@@ -175,7 +175,7 @@ export function Query<Q extends Node>(props: QueryProps<Q>): JSX.Element | null
         component = <WebVitalsPathBreakdown query={query} cachedResults={props.cachedResults} context={queryContext} />
     } else if (isHogQuery(query)) {
         component = <HogDebug query={query} setQuery={setQuery as (query: any) => void} queryKey={String(uniqueKey)} />
-    } else if (isEventsHeatMapQuery(query)) {
+    } else if (isCalendarHeatmapQuery(query)) {
         component = <WebActiveHoursHeatmap query={query} context={queryContext} cachedResults={props.cachedResults} />
```

Comment:
Maybe you want to update the component name as well, but no need to do it on this PR. (I did not check the other one so far, so this could be done already)